sdk/deviceupdate/iot-device-update/package.json

```diff
@@ -82,11 +82,11 @@
     "build:samples": "echo Obsolete.",
     "build:test": "echo skip",
     "build": "npm run clean && tsc -p . && rollup -c 2>&1 && api-extractor run --local",
-    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
```

Comment:
In order to fix the CI, it was necessary to remove the path `"test/**/*.ts"` from the `format` and `check-format` scripts.
This is due to `iot-device-update` not having a test folder, which throws an error when running any of those scripts.